### DISCUSSION TITLE
Is there Krypton Suite for Delphi ?

### DISCUSSION THREAD

**Discussion by izac3d** [Krypton-Suite/Standard-Toolkit#1467] [Q&A] [answered]
I read in the help example syntax for Delphi. Is there Delphi edition from Krypton ?

**Comment by PWagner1** [comment, marked as answer]
Hi @izac3d 

The only platforms/languages that are supported are .NET Framework/.NET with C# & Visual Basic .NET. Delphi is not supported.

**Comment by giduac** [comment]
Hi @Wagnerp,
Looks like this can be closed...
